dxchange/reader.py

```diff
@@ -651,40 +654,71 @@ def read_dx_meta(file_name, label1='/measurement/', label2='/process/') :
     """
     meta = {}
 
-    fp = h5py.File(file_name, 'r') 
+    fp = h5py.File(file_name, 'r')
     read_hdf5_item_structure(meta, fp, file_name, label1, label2)
     fp.close()
 
     return meta
 
 
+def create_standard_dx_meta(file_name):
+    """
+    Create meta data dict for files written in standard dxfile format
+
+    :param file_name: file name of the HDF5 input file that follows dxfile standard file structure
+    :return meta: meta data dictionary using standard names for keys
+    """
+
+    h5file = h5py.File(file_name, 'r')
```

Comment:
This opens the file but leaves it open. 

Since h5py.File() is a python context manager, you could add the following and indent your code:
```
with h5py.File(file_name, 'r') as h5file:
  meta = dict()....
```

Reply:
thanks @dylanmcreynolds. I am testing a new reader (see my comment below) and already corrected this [here](https://github.com/xray-imaging/meta/blob/df5222c4ab13b32e2ec0bc3ecfccbff12a3b3a0d/meta/fileio.py#L41). I will make sure to bring this back here

Reply:
this is now resolved in the latest version